mcp_server/server.py

```
@@ -0,0 +1,966 @@
+            "weights": explanation.weights,
+            "explanations": explanation.explanations,
+        },
+    }
+
+
+@mcp.tool()
+@handle_mcp_error
+async def register_fitness_function(
+    name: str,
+    weight: float = 1.0,
+    description: Optional[str] = None,
```

Ruff (0.14.10)

Unused function argument: `description`

(ARG001)